ComicsWebApp/Models/ComicsViewModel.cs

```diff
@@ -0,0 +1,13 @@
+﻿using Microsoft.AspNetCore.Mvc.Rendering;
+
+namespace ComicsWebApp.Models
+{
```

Comment:
I think we should have ComicsAddEditModel (that we will use for Add/Edit) and ComicsViewModel that we will use only to show info on the page